pkg/threescale/threescale.go

```diff
@@ -368,7 +369,9 @@ func NewThreescale(addr string, client *http.Client, conf *AdapterConfig) (Serve
 
 	log.Infof("Threescale Istio Adapter is listening on \"%v\"\n", s.Addr())
 
-	s.server = grpc.NewServer()
+	s.server = grpc.NewServer(grpc.KeepaliveParams(keepalive.ServerParameters{
+		MaxConnectionAge: 1 * time.Minute,
```

Comment:
I think it would be useful for us to be able to play with this setting dynamically - maybe set as an ENV parameter?

Reply:
+1